src/docs/product/integrations/integration-platform/webhooks.mdx

```diff
@@ -51,15 +51,20 @@ function verifySignature(request, secret = "") {
 ```
 
 ```python
-    body = json.dumps(request.body)
-    expected = hmac.new(
-        key=client_secret.encode('utf-8'),
-        msg=body,
-        digestmod=sha256,
-    ).hexdigest()
-
-    if expected != request.headers['Sentry-Hook-Signature']:
-            raise UnauthorizedError
+import hashlib
+import hmac
+import json
+
+expected = request.headers['sentry-hook-signature']
+body = json.dumps(request.body)
+digest = hmac.new(
+    key=client_secret.encode('utf-8'),
+    msg=body,
+    digestmod=hashlib.sha256,
+).hexdigest()
+
+if digest != expected:
```

Comment:
IMO think it'd be more effective to change the variable name to `expected_digest` and declare it after L60.

Reply:
> after L60
I meant after `digest`
```python
digest = ...
expected_digest = ...
```

Reply:
I prefer to collect all params from `request` together at the top of the function.